### PR TITLE
Fix docs URL not using https.

### PR DESCRIPTION
**Pull request by ghost**
I left off links in `docs/sources/release-notes.md` as it didn't seem right, and there's one left in `docs/mkdocs.yml`. Should I replace them as well?

### CODE REVIEW

**Comment by thaJeztah** [comment]
> Should I replace them as well?

I'd say; go for it, I don't see a reason for them to be excluded.

Thanks for doing this!

**Comment by eolamey** [comment]
Sure, no problem, this is really nothing.

I updated the PR. There's one link left in `docs/sources/installation/mac.md`, but it's the output of `docker run hello-world`, so I left it as-is (the `hello-world` image should probably be updated, though).

**Comment by thaJeztah** [comment]
I noticed that the other day :) So created a PR https://github.com/docker-library/hello-world/pull/5

thanks for this! I'll check as soon as I'm behind my computer.

**Comment by thaJeztah** [comment]
Yup, LGTM!

ping @moxiegirl (perhaps also for cherry-picking)

File: docs/sources/docker-hub-enterprise/quick-start.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ You should be able to complete this guide in about thirty minutes.
 > **Note:** This guide assumes you are familiar with basic Docker concepts such
 > as images, containers, and registries. If you need to learn more about Docker
 > fundamentals, please consult the
-> [Docker user guide](http://docs.docker.com/userguide/).
+> [Docker user guide](https://docs.docker.com/userguide/).
```
Comment (moxiegirl) [inline docs/sources/docker-hub-enterprise/quick-start.md]: This file is copied from one of our private repos which you don't have access to.  The change actually has to be made there or else the next copy over will overwrite your change. ping @SvenDowideit @fredlf for the fix on dhe repo.

File: docs/sources/docker-hub-enterprise/configuration.md [hunk 2]
```diff
@@ -258,7 +258,7 @@ API.
 * *Yaml configuration file*: This file (`/usr/local/etc/dhe/storage.yml`) is
 used to configure the image storage services. The editable text of the file is
 displayed in the dialog box. The schema of this file is identical to that used
-by the [Registry 2.0](http://docs.docker.com/registry/configuration/).
+by the [Registry 2.0](https://docs.docker.com/registry/configuration/).
```
Comment (moxiegirl) [inline docs/sources/docker-hub-enterprise/configuration.md]: This file is copied from one of our private repos which you don't have access to.  The change actually has to be made there or else the next copy over will overwrite your change. ping @SvenDowideit @fredlf for the fix on dhe repo.

**Comment by moxiegirl** [comment]
@eolamey Thanks for the fixes.  LGTM.  RE the DHE links we'll pull your changes but as my notes inline indicate they may creep back in if the change isn't made on the private repo.

**Comment by moxiegirl** [comment]
ping @jfrazelle for the cherry pick.

**Comment by jfrazelle** [comment]
cherrypicked

**Comment by fredlf** [comment]
Thanks all, will fix in DHE repo.

**Pull request merged**